Benutzerdoku/Benutzerdoku.tex

```diff
@@ -0,0 +1,11 @@
+\documentclass{scrartcl}
+
+\usepackage[ngerman]{babel}
+\usepackage{graphicx}
+\usepackage{wrapfig}
+\usepackage[section]{placeins}
+
+\begin{document}
+	\input{Admin}
```

Comment:
Hier könnte noch ein Inhalts- und evtl. Abbildungsverzeichnis rein. Geht ganz einfach mit 
`\tableofcontents` und
`listoffigures`
Es kann sein, dass LaTeX dabei Dateien generiert, die nicht mit auf git landen sollten. Diese solltest du dann in `.gitignore` einfügen